### ISSUE TITLE
GPT-J rotary embeddings cause logits to be returned as FakeTensors when scan=True

### ISSUE THREAD

**Issue by ericwtodd** [ndif-team/nnsight#162]
```
import nnsight
import torch

model_name = 'EleutherAI/gpt-j-6b'

model = nnsight.LanguageModel(model_name, device_map='cuda', dispatch=True)

with model.trace('The Eiffel Tower is located in the city of'):
    out = model.output.save()
    
print(out.logits)
```

In the above code, the logits are returned as FakeTensors since `scan=True`  in the trace context by default. It looks like it's because of the GPT-J rotary embeddings [here](https://github.com/huggingface/transformers/blob/main/src/transformers/models/gptj/modeling_gptj.py#L208).

@JadenFiotto-Kaufman pointed out to me two current solutions. Either setting ` torch.jit.is_tracing = lambda : True`, or just making sure  `scan` & `validate` are set to `False`.

**Comment by JadenFiotto-Kaufman** [comment]
This is "fixed" on dev by forcing `torch.jit.is_tracing` to be true during scanning. Will be in next release.
